pkg/oras/discover.go

```diff
@@ -2,6 +2,7 @@ package oras
 
```

Comment:
Given that this is a prototype branch I don't have concerns on this change. For the `oras-go` would like to ensure that discover has some tests.

Reply:
Actually I wrote tests already in oras-go so I can port them over here